appserver/website-bootstrap/network.html

```diff
@@ -46,6 +47,156 @@ <h3>Analyze Network</h3>
             <div class="col-12 col-md-10">
                 <header class="page-header header container-fluid">
                     <svg id="network"></svg>
+					<script>
+
+						var width = 900, height = 600;
+
+						var svg = d3.select("#network")
+							.attr("width", width)
+							.attr("height", height);
+
+						var simulation = d3.forceSimulation()
+							.force("link", d3.forceLink().id(function(d) { return d.id; }))
+							.force("charge", d3.forceManyBody()
+								.strength(-300)
+								.theta(0.8)
+								.distanceMax(150)
+							)
+							.force("center", d3.forceCenter(width / 2, height / 2));
+
+						const graphData = {
+							"nodes": [
+								{"id": "n1"},
+								{"id": "n2"},
+								{"id": "n3"},
+								{"id": "n4"},
+								{"id": "n5"},
+								{"id": "n6"},
+								{"id": "n7"},
+								{"id": "n8"},
+								{"id": "n9"},
+								{"id": "n10"},
+								{"id": "n11"},
+								{"id": "n12"},
+								{"id": "n13"},
+								{"id": "n14"},
+								{"id": "n15"},
+								{"id": "n16"},
+								{"id": "n17"},
+								{"id": "n18"},
+								{"id": "n19"},
+								{"id": "n20"}
+							],
+							"links": [
+								{"source": "n1", "target": "n2", "value": 1},
+								{"source": "n1", "target": "n12", "value": 3},
+								{"source": "n1", "target": "n13", "value": 2},
+								{"source": "n1", "target": "n11", "value": 30},
+								{"source": "n13", "target": "n2", "value": 1},
+								{"source": "n15", "target": "n2", "value": 12},
+								{"source": "n4", "target": "n20", "value": 1},
+								{"source": "n17", "target": "n2", "value": 10},
+								{"source": "n18", "target": "n20", "value": 1},
+								{"source": "n19", "target": "n12", "value": 1},
+								{"source": "n19", "target": "n9", "value": 1},
+								{"source": "n11", "target": "n17", "value": 19},
+								{"source": "n9", "target": "n20", "value": 1},
+								{"source": "n17", "target": "n5", "value": 11},
+								{"source": "n1", "target": "n14", "value": 3},
+								{"source": "n1", "target": "n15", "value": 7},
+								{"source": "n1", "target": "n16", "value": 12},
+								{"source": "n2", "target": "n4", "value": 7},
+								{"source": "n3", "target": "n5", "value": 9},
+								{"source": "n2", "target": "n10", "value": 7},
+								{"source": "n3", "target": "n9", "value": 7},
+								{"source": "n4", "target": "n8", "value": 20},
+								{"source": "n4", "target": "n7", "value": 15},
+								{"source": "n8", "target": "n1", "value": 1},
+								{"source": "n6", "target": "n1", "value": 1}
+							]
+						}
+  
+						function run(graphData) {
```

Comment:
I might be misunderstanding Javascript syntax, but isn't this line declaring a new function which has a parameter named 'graphData'? My concern had been that the variable which was declared before the function definition had the same name as the parameter of that function.
While in this case the existing variable was then simply passed to that function as an argument (for the parameter of the same name), my concern was that the parameter & the already-existing variable having the same name might cause confusion if this code is expanded on later.

Reply:
ohh my bad I misread what you sent the first time yeah I got it

Reply:
No problem. Thanks!